projects/bp-gallery/environments/.env.development

```diff
@@ -1,3 +1,3 @@
-VITE_REACT_APP_API_BASE=https://bilder-dev.bad-harzburg-stiftung.de/api
+VITE_REACT_APP_API_BASE=http://127.0.0.1:9000
```

Comment:
shouldn't be commited, use .env.development.local instead

Reply:
ah, das gibts! danke ^^